lib/susedistribution.pm

```diff
@@ -74,6 +74,7 @@ sub init_cmd {
       entiredisk alt-e
       guidedsetup alt-g
       rescandevices alt-e
+      exp_part_finish alt-f
```

Comment:
sorry I am totally out of concept behind this $cmd but what the point to define something in one place to redifine it in another ?

Reply:
Here we define default key and for storage-ng these are different, so we redefine it, but in the code we can use same hash key without conditional switches.

Reply:
and this is the way to go for testing localization eventually :)

Reply:
ok